index.md

```diff
@@ -1,13 +1,16 @@
 <strong>
-Unfortunately, [orocos.org](http://www.orocos.org/) was down since mid August and we have not found a quick and easy way to fix it. It is planned to create a new website and also restore the contents of the Wiki that are still up-to-date.
-</strong> 
- 
+The official site of [orocos.org](https://orocos.org/) is back online after
+an upgrade.
+Orocos related [documentation](https://docs.orocos.org/) now centralizes all
+the documentation related with the Orocos Project.
```

Comment:
"Orocos related documentation now centralizes all the documentation related with the Orocos Project."? Does not make sense to me.

```suggestion
[docs.orocos.org](https://docs.orocos.org/) now centralizes all the documentation related to the Orocos Project.
```